src/main/java/uk/gov/hmcts/reform/wataskmanagementapi/controllers/advice/ErrorMessage.java

```diff
@@ -0,0 +1,24 @@
+package uk.gov.hmcts.reform.wataskmanagementapi.controllers.advice;
+
+
+import lombok.Data;
+import org.springframework.http.HttpStatus;
+
+import java.time.ZonedDateTime;
+import java.time.format.DateTimeFormatter;
+
+@Data
+public class ErrorMessage {
+
+    private String timestamp;
+    private String error;
+    private int status;
+    private String message;
+
+    public ErrorMessage(Exception ex, HttpStatus error) {
+        this.timestamp = ZonedDateTime.now().format(DateTimeFormatter.ISO_INSTANT);
```

Comment:
Is this the correct Format within `hmcts` can't remember but this is usually the standard:  https://tools.ietf.org/html/rfc3339#section-5.6.

It may be worth creating a class that always gives you the correct format?